nautobot/dcim/urls.py

```diff
@@ -46,28 +45,9 @@
 router.register("module-bay-templates", views.ModuleBayTemplateUIViewSet)
 router.register("module-types", views.ModuleTypeUIViewSet)
 router.register("virtual-device-contexts", views.VirtualDeviceContextUIViewSet)
+router.register("location-types", views.LocationTypeUIViewSet)
```

Comment:
Can we alphabetize this list? I don't see any logical order to the current code.